tests/common.py

```diff
@@ -25,12 +25,33 @@ def get_test_case(path: str, obj=None, raw=False):
 
     if obj is None:
         return dict_data
-    return obj.from_dict(dict_data)
+
+    if obj is DetectedIssuesResponse and type(dict_data) is list:
+        return obj(issue_reports=dict_data)
+    else:
+        return obj(**dict_data)
 
 
 AST = get_test_case("testdata/test-ast.json")
 
 
+@contextmanager
+def mock_faas_context():
+    # with patch("mythx_cli.fuzz.rpc.RPCClient.get_all_blocks", new=) as get_all_blocks_patch, patch(
+    #     "mythx_cli.fuzz.rpc.RPCClient.contract_exists"
+    # ) as contract_exists_patch:
+    #     print("-->patching blocks")
+    #     get_all_blocks_patch.return_value = get_test_case("testdata/ganache-all-blocks.json")
+    #     contract_exists_patch.return_value = True
```

Comment:
We better remove commented code fragment before merging.

Reply:
Yup! :D